index.md

```diff
@@ -37,3 +37,6 @@ Your Pages site will use the layout and styles from the Jekyll theme you have se
 ### Support or Contact
 
 Having trouble with Pages? Check out our [documentation](https://docs.github.com/categories/github-pages-basics/) or [contact support](https://support.github.com/contact) and we’ll help you sort it out.
+dzis jest piatek
+zosia
```

Comment:
this is out of context, please remove it

Reply:
ok, zosia removed